web/modules/custom/joinup_core/joinup_core.routing.yml

```diff
@@ -54,3 +54,13 @@ joinup_core.id_redirect:
     _controller: '\Drupal\joinup_core\Controller\IdRedirect::redirectToRdfEntity'
   requirements:
     _access: 'TRUE'
+
+joinup_core.go_to_rdf_entity_page:
+  path: '/rdf'
+  defaults:
+    _form: 'Drupal\joinup_core\Form\GoToRdfEntityPage'
+    _title: 'Go to RDF entity page'
+  requirements:
+    _permission: 'administer nodes'
```

Comment:
This should depend either on 'administer rdf entity' which is defined in the Rdf entity annotation, or a new permission that is specific to this page.

Reply:
The idea was to provide a handy tool, an "easy to type" thing. If we put there a long URL will not help. Right now, on our project, this URL is not used. If it will be in the future, we can change it.

Reply:
Yeah, that was my first thought. But I realised that is not used anywhere and no role is granted with that permission. Ideally we would need a custom permission, something like "administer content" that should cover nodes and RDF entities. I picked up "administer nodes" because that is assigned to "administrator" role.